doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -66,7 +66,7 @@ local html = import 'html.libsonnet';
         {
           name: 'get',
           params: ['o', 'f', 'default=null', 'inc_hidden=true'],
-          availableSince: 'upcoming',
+          availableSince: '0.18.0',
```

Comment:
These manual HTML edits were introduced in https://github.com/google/jsonnet/pull/957.